### PR TITLE
Place parsed ini file in results directory

### PR DESCRIPTION
**Pull request by titodalcanton**
This completes my earlier commit, getting the config file to show in HTML report pages. Alex, I'm not sure my handling of the file is the way it's supposed to be, but I couldn't find a better functionality in pycbc.workflow. Anyway, it seems to do the job.

### CODE REVIEW

File: bin/hdfcoinc/pycbc_make_hdf_coinc_workflow [hunk 1]
```diff
@@ -141,6 +144,17 @@ if len(inj_files) > 0:
     wf.make_sensitivity_plot(workflow, found_inj, 'plots/sensitivity', 
                                             tags=['ALLINJ'])
 
+# save global config file to results directory
+ini_file_path = 'plots/configuration.ini'
+with open(ini_file_path, 'wb') as ini_fh:
+    container.cp.write(ini_fh)
```
Comment (ahnitz) [inline bin/hdfcoinc/pycbc_make_hdf_coinc_workflow]: Tito, 

I'm not sure why you need the following lines. It looks like you've already written the configuration file to the plots directory (which we should rename at some point...). That should be all you need for it to show up in the results page.

**Comment by titodalcanton** [comment]
Hi Alex,

I first tried to manually copy the ini file to the plots directory while the workflow was running. When the workflow completed I found that somehow the file got deleted before the HTML report was written. So I thought that maybe Pegasus required to know about every file in the final directory, that's why I add that code. If you're sure that's not the case, I'll remove it and keep it simple.

**Comment by ahnitz** [comment]
Ok, I see. I don't know of any reason for it to have been deleted. We have lots of files like this, such as segment files / log files.

**Comment by titodalcanton** [comment]
But segment files are always the output of some job in the workflow right? So the workflow "knows" about them being there.

**Comment by ahnitz** [comment]
It doesn't know about the shell scripts or log files though, plus there is
no cleanup job for them.

**Comment by titodalcanton** [comment]
Ok, then I'll clean the patch up and merge it.

**Pull request merged**